### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by lucascaro** [wiziple/gatsby-plugin-intl#168]
Hi there, thank you so much for creating and maintaining this project!

Looking at the amount of non-responded issues and PRs I'm wondering what's the official status of this project? seems that the only commits in a while are small documentation updates...
Has this project been abandoned?
Is it looking for maintainers?
Any way the community can help move this forward?

**Comment by shi-rudo** [comment]
would like to get a statement on this as well.

**Comment by AleC77** [comment]
Yes it is important to know. We're moving at Gatsby 3 and it says:
warn Plugin gatsby-plugin-intl is not compatible with your gatsby version 3.0.3 - It requires gatsby@^2.0.0

**Comment by giacomo-remago** [comment]
Any update?

**Comment by shi-rudo** [comment]
no answer is also an answer

**Comment by shi-rudo** [comment]
a working fork!

https://www.npmjs.com/package/gatsby-plugin-react-intl

**Comment by M4ss1ck** [comment]
well, in that plugin readme.md you can read "this plugin will no longer be actively maintained"

I wonder what the new tool for i18n in Gatsby is

**Comment by shi-rudo** [comment]
> 

I can highly recommend https://github.com/microapps/gatsby-plugin-react-i18next as an alternative

**Comment by openscript** [comment]
I've also been working on an alternative for a while: https://github.com/openscript-ch/gatsby-plugin-i18n-l10n
